types/user/iterations.pp

```diff
@@ -0,0 +1,4 @@
+type Accounts::User::Iterations = Variant[
+  Integer[1,],
+  Pattern[/^[1-9]\d*$/]
```

Comment:
Prefer `\A` and `\z` to `^` and `$`: if the value is a multi=line string, you will accept invalid data (e.g. `"hello\n123\nworld\n").

Reply:
Implemented your suggested changes.